dmocks/object_mock.d

```diff
@@ -1,5 +1,7 @@
 module dmocks.object_mock;
 
+import core.vararg;
```

Comment:
Doesn't the import already there:
https://github.com/funkwerk/DMocks-revived/blob/3176683a5c57d435ede0263985066e989bcc309f/dmocks/object_mock.d#L9

Reply:
Oops. Guess we just need to go up to 1.0.2.